test/visualisation/latexify.jl

```diff
@@ -48,68 +50,68 @@ let
         (d1,d2,d3,d4,d5,d6), (X1,X2,X3,X4,X5,X6)  ⟶ ∅
     end
 
-    # Latexify.@generate_test latexify(rn)
-    @test_broken latexify(rn; expand_functions = false) == replace(
```

Comment:
just a simple error, `expand_functions = false` tested twice.
(`true` is the default, so don't really need it in the next one, but specified it anyway)